argos_system.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+DOCKER_COMPOSE_FILE="docker-compose-prod.yml"
+
+log() {
+  echo -e "\033[1;32m$1\033[0m"
+}
+
+error() {
+  echo -e "\033[1;31mError: $1\033[0m"
+  exit 1
+}
+
+if ! command -v docker &> /dev/null; then
+  error "Docker is not installed. Please install it first."
+fi
+
+if ! command -v docker-compose &> /dev/null; then
+  error "Docker Compose is not installed. Please install it first."
+fi
+
+up() {
+  log "Lifting containers with $DOCKER_COMPOSE_FILE..."
+  docker-compose -f $DOCKER_COMPOSE_FILE pull || error "Error when obtaining images."
+  docker-compose -f $DOCKER_COMPOSE_FILE --profile production up -d --build || error "Error when lifting the containers."
+  log "Containers successfully lifted."
+}
+
+down() {
```

Comment:
When running this command, I had this output.

![image](https://github.com/user-attachments/assets/93f4bda5-985c-46de-a382-1fc606bc46ba)

So my question is, does it should only delete one container?